translations/mod-record-specifications/en.json

```diff
@@ -1,6 +1,8 @@
 {
   "specification.field.tag.duplicate": "The 'tag' must be unique.",
   "specification.field.label.duplicate": "The 'label' must be unique.",
+  "field.indicator.order.duplicate": "Can only have one validation rule per MARC field indicator/order.",
+  "indicator.code.code.duplicate": "Can only have one validation rule per MARC field indicator/code.",
```

Comment:
Could you please follow the same format as for tag? 
And please remove `specification.field.label.duplicate` key (I forgot to do it after the requirements changed)